sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActionResult.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.ai.textanalytics.implementation.TextAnalyticsActionResultPropertiesHelper;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.OffsetDateTime;
+import java.util.Locale;
+
+/**
+ * The {@link TextAnalyticsActionResult} model.
+ */
+public class TextAnalyticsActionResult {
+    private final ClientLogger logger = new ClientLogger(TextAnalyticsActionResult.class);
+
+    private OffsetDateTime completedAt;
+    private TextAnalyticsError error;
+    private boolean isError;
+
+    static {
+        TextAnalyticsActionResultPropertiesHelper.setAccessor(
+            new TextAnalyticsActionResultPropertiesHelper.TextAnalyticsActionResultAccessor() {
+                @Override
+                public void setCompletedAt(TextAnalyticsActionResult actionsResult, OffsetDateTime completedAt) {
+                    actionsResult.setCompletedAt(completedAt);
+                }
+
+                @Override
+                public void setError(TextAnalyticsActionResult actionResult, TextAnalyticsError error) {
+                    actionResult.setError(error);
+                }
+
+                @Override
+                public void setIsError(TextAnalyticsActionResult actionResult, boolean isError) {
+                    actionResult.setIsError(isError);
+                }
+            });
+    }
+
+    /**
+     * Gets the completed time of an entities recognition action.
```

Comment:
```suggestion
     * Gets the time when the aciton was completed.
```

or something like that?